pricejar/base/templates/base/home.html

```diff
@@ -2,8 +2,370 @@
 
```

Comment:
You need to inherit navbar and footer from base.html. Also load your static files with {% load static %}.